XamlControlsGallery/App.xaml

```diff
@@ -59,6 +59,8 @@
             <Thickness x:Key="PageHeaderDefaultPadding">12,28,12,0</Thickness>
             <Thickness x:Key="PageHeaderMinimalPadding">-4,0,12,0</Thickness>
 
+            <Thickness x:Key="ControlElementScreenshotModePadding">67</Thickness>
```

Comment:
Is there a specific reason that 67 is being used here or does it just look good?

Reply:
It's the value I was given by the documentation team.

Reply:
Alright, I see. Thanks for the clarification.